sql/core/src/test/scala/org/apache/spark/sql/InjectRuntimeFilterSuite.scala

```diff
@@ -209,11 +209,13 @@ class InjectRuntimeFilterSuite extends QueryTest with SQLTestUtils with SharedSp
     // `MergeScalarSubqueries` can duplicate subqueries in the optimized plan and would make testing
     // complicated.
     conf.setConfString(SQLConf.OPTIMIZER_EXCLUDED_RULES.key, MergeScalarSubqueries.ruleName)
+    conf.setConf(SQLConf.CBO_ENABLED, false)
```

Comment:
To fix the test error:
<img width="1195" alt="image" src="https://user-images.githubusercontent.com/5399861/178422514-d5bfdac8-5518-40d3-b8ae-6c6571b351e4.png">

Reply:
Seems this line is to keep consistent with the previous behavior, but should we add some TODO for this change? For example, spilt this Suites into `withCBOEnabled` and `withCBODisabled`?

I found many suites in this class will fail when `CBOEnabled`.